packages/go_router/doc/navigation.md

```diff
@@ -0,0 +1,56 @@
+There are many ways to navigate between destinations in your app.
+
+## Go directly to a destination
+Navigating to a destination in GoRouter will replace the current stack of screens with the screens configured to be displayed
+for the destination route. To change to a new screen, call `context.go()` with a URL:
+
+```
+context.go('/users/123')
+```
+
+This is shorthand for calling `GoRouter.of(context).go('/users/123)`.
```

Comment:
Could you add a query parameter example too?

Reply:
Sure, I think the current best-practice is to use the URI class:

```
Uri(path: '/users/123', queryParameters: {'filter': 'abc'}).toString()
```